subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,93 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
+are organized so that it continue even after members go away.
+
+## Terminology
+
+-   **Debriefing session**: is an online meeting
+    where instructors could share experience about the workshops
+    they taught.
+-   **Facilitator**: is one member of the Mentoring Subcommittee
+    that attend the debriefing session to coordinate it
+    and write one blog post about it.
+
+## We Need a Debriefing Session
+
+1.  SCF is a community based project
+    and the community need opportunity to interact.
+
+2.  SCF believe in jugyokenkyu, or "lesson study",
```

Comment:
Same as above (add DC)? The Instructor Training and Instructor community is shared between DC and SWC and so both subscribe to this philosophy (it's just making it explicit).